infra/app/storage.bicep

```diff
@@ -0,0 +1,25 @@
+param name string
```

Comment:
why not infra/core?

Reply:
We need to create `fileshares` with `StorageAccount` as the parent, but `core/storage/storage-account.bicep` does not support this part. Based on your [comment](https://github.com/jongio/chat-with-your-data-solution-accelerator/pull/2#discussion_r1449306153) in another sample, updated the `core/storage/storage-account.bicep` file to support this.